database/tbcd/database.go

```diff
@@ -19,6 +20,26 @@ import (
 	"github.com/hemilabs/heminetwork/database"
 )
 
+type InsertType int
+
+const (
+	ITInvalid     InsertType = 0 // Invalid insert
+	ITChainExtend InsertType = 1 // Normal insert, does not require further action.
+	ITChainFork   InsertType = 2 // Chain forked, unwind and rewind indexes.
+	ITForkExtend  InsertType = 3 // Extended a fork, does not require further action.
+)
+
+var ITStrings = map[InsertType]string{
```

Comment:
Is there a reason to export this variable?

Reply:
No. Fixed.